### DISCUSSION TITLE
请问如何隐藏.desktop文件？

### DISCUSSION THREAD

**Discussion by Huayuluoshi** [fcitx/fcitx5#1179] [General]
### Summary

作者你好，我现在使用fcitx5,觉得不需要那些desktop文件；但是看过一个issue说有手动启动的需求，那么我有些好奇怎么配置才可以让desktop不显示？谢谢！

**Comment by wengxt** [comment]
https://specifications.freedesktop.org/desktop-entry-spec/latest/recognized-keys.html 
NoDisplay

如果你用桌面环境的话，一般可以从桌面配置
